tutorials/plot_ecog.py

```diff
@@ -46,7 +46,8 @@
 #           do not align to the cortical surface perfectly.
 
 subjects_dir = mne.datasets.sample.data_path() + '/subjects'
-fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
+fig = plot_alignment(info, trans=None, subject='sample',
```

Comment:
I actually don't like `'auto'`, it's too magical, and when actually using `trans` in other places (like fwd) you actually need to specify the path anyway. So I'd like to remove it.

If we do, we could make `None` the default instead. That's what we do most other places.